src/test/analyzer_test.ts

```diff
@@ -12,15 +12,17 @@
  * http://polymer.github.io/PATENTS.txt
  */
 
-'use strict';
+/// <reference path="../../node_modules/@types/mocha/index.d.ts" />
```

Comment:
```
polymer-build@0.6.0-alpha.3 /Users/fschott/Code/polymer-build
└── typescript@2.1.4 
```

Hmm, after some googling it looks like you can pass an option to your tsconfig to reference mocha automatically: http://stackoverflow.com/a/40197672 This means TypeScript will also mistakenly think `suite` and `test` are available in your library code as well, but I don't think that's a problem we'd ever encounter normally. 

I'm fine with adding this to our `tsconfig` and removing the explicit reference, if there aren't any objections? We could do the same in polymer-analyzer as well.